<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font-family : Tahoma;
	min-height : 25px;
}
QPushButton#b_screenshot{
	min-height:50px;
}
QWidget#widget{
	background-color: rgb(250, 250, 250);
	border : 1px solid gray;
}
QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(232, 232, 232, 255), stop:0.329545 rgba(243, 243, 243, 255), stop:0.994318 rgba(221, 221, 221, 255));
	border-radius : 3px;
	border-color : gray;
	border-style : solid;
	border-width : 1px;
}
QPushButton:hover{
	background-color :  qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(212, 212, 212, 255), stop:0.329545 rgba(223, 223, 223, 255), stop:0.994318 rgba(201, 201, 201, 255));
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QPushButton" name="b_exit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string>exit</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/door_out.png</normaloff>:/door_out.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="b_setting">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string>setting</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/settings.png</normaloff>:/settings.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="b_screenshot">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>52</height>
          </size>
         </property>
         <property name="text">
          <string>screenshot</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/camera.png</normaloff>:/camera.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
